<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rotable</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="_tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <widget class="QWidget" name="_tabOrders">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/rotable/toggle_log.png</normaloff>:/rotable/toggle_log.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Orders</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="OrderListView" name="m_list_orders_ptr"/>
        </item>
        <item>
         <widget class="QCheckBox" name="m_checkbox_list_completed_orders_ptr">
          <property name="text">
           <string>List completed orders</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_tabDatabase">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/rotable/db.png</normaloff>:/rotable/db.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Database</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QToolButton" name="_toolButtonDbExport">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Export</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/rotable/db_update.png</normaloff>:/rotable/db_update.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextUnderIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="_toolButtonDbImport">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Import</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/rotable/db_comit.png</normaloff>:/rotable/db_comit.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextUnderIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="_toolButtonDbReset">
               <property name="text">
                <string>Clear</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/rotable/db_clear.png</normaloff>:/rotable/db_clear.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextUnderIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="_toolButtonAddCategory">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Add Category</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/rotable/add.png</normaloff>:/rotable/add.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextUnderIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="_toolButtonAddProduct">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Add Product</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/rotable/add.png</normaloff>:/rotable/add.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextUnderIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="_toolButtonRename">
               <property name="text">
                <string>Rename</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/rotable/edit.png</normaloff>:/rotable/edit.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextUnderIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="_toolButtonRemove">
               <property name="text">
                <string>Remove</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/rotable/remove.png</normaloff>:/rotable/remove.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextUnderIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSplitter" name="_splitterDb">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="CategoryListView" name="_listViewCategories">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="viewMode">
               <enum>QListView::ListMode</enum>
              </property>
              <property name="uniformItemSizes">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="ProductTableView" name="_tableViewProducts">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_tabSales">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/rotable/office-chart-area-stacked.png</normaloff>:/rotable/office-chart-area-stacked.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Sales</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="_toolButtonExport">
            <property name="text">
             <string>Export</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/resources.qrc">
              <normaloff>:/rotable/pdf.png</normaloff>:/rotable/pdf.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Statistik_Plot" name="d_plot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_tabAds">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/rotable/video-x-generic-2.png</normaloff>:/rotable/video-x-generic-2.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Advertising</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="_tabServerDebug">
       <attribute name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/rotable/toggle_log.png</normaloff>:/rotable/toggle_log.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Server Debug</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="_toolButtonStartDebugListening">
            <property name="text">
             <string>Start Listening</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/resources.qrc">
              <normaloff>:/rotable/network-connect.png</normaloff>:/rotable/network-connect.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="_toolButtonStopDebugListening">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Stop Listening</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/resources.qrc">
              <normaloff>:/rotable/network-disconnect.png</normaloff>:/rotable/network-disconnect.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="_toolButtonClearDebugListening">
            <property name="text">
             <string>Clear</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/resources.qrc">
              <normaloff>:/rotable/db_clear.png</normaloff>:/rotable/db_clear.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="_textEditServerLog">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menubar_ptr">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="m_toolbar_ptr">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="_actionConnect"/>
   <addaction name="_actionDisconnect"/>
  </widget>
  <widget class="QStatusBar" name="_statusBar"/>
  <action name="_actionConnect">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/rotable/network-connect.png</normaloff>:/rotable/network-connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to server</string>
   </property>
  </action>
  <action name="_actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/rotable/network-disconnect.png</normaloff>:/rotable/network-disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from server</string>
   </property>
  </action>
  <action name="action_export_db">
   <property name="text">
    <string>Export Database</string>
   </property>
   <property name="toolTip">
    <string>Export the entire database</string>
   </property>
  </action>
  <action name="action_add_product">
   <property name="icon">
    <iconset>
     <normaloff>:/rotable/edit-add-2.png</normaloff>:/rotable/edit-add-2.png</iconset>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
   <property name="toolTip">
    <string>Add a product</string>
   </property>
  </action>
  <action name="action_add_category">
   <property name="icon">
    <iconset>
     <normaloff>:/rotable/edit-add-2.png</normaloff>:/rotable/edit-add-2.png</iconset>
   </property>
   <property name="text">
    <string>Add Category</string>
   </property>
   <property name="toolTip">
    <string>Add a product category</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OrderListView</class>
   <extends>QListView</extends>
   <header>orderlistview.h</header>
  </customwidget>
  <customwidget>
   <class>CategoryListView</class>
   <extends>QListView</extends>
   <header>categorylistview.h</header>
  </customwidget>
  <customwidget>
   <class>ProductTableView</class>
   <extends>QTableView</extends>
   <header>producttableview.h</header>
  </customwidget>
  <customwidget>
   <class>Statistik_Plot</class>
   <extends>QFrame</extends>
   <header>statistik_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_list_orders_ptr</tabstop>
  <tabstop>m_checkbox_list_completed_orders_ptr</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
